web/e2e-tests/stream_create.test.ts

```diff
@@ -81,6 +81,14 @@ async function create_stream(page: Page): Promise<void> {
         stream_description: "Everything Puppeteer",
     });
     await page.click("form#stream_creation_form .finalize_create_stream");
+    // an explanatory modal is shown for the first stream created
+    await page.waitForSelector("modal__btn dialog_submit_button");
+    await page.click("modal__btn dialog_submit_button");
+    await page.waitForSelector(".message-header-stream-settings-button.tippy-zulip-tooltip", {
```

Comment:
Can we remove `tipply-zulip-tooltip` class from selector here?

Reply:
Sure, I've removed it.